Pool/spec.ts

```diff
@@ -83,6 +84,11 @@ class Pool extends LiveObject {
         this.adminRoleId = adminRoleId;
 
         this.createdAt = this.blockTimestamp;
+
+        const contractGroupName = await getStrategyGroup(this.chainId, this.strategy)
+        if (contractGroupName) {
+            this.addContractToGroup(this.address, contractGroupName)
+        }
```

Comment:
@whittlbc might need a hand to validate if this makes sense

Reply:
I think you probably mean to add the strategy address to the group, but other than that makes sense to me!:
```typescript
this.addContractToGroup(this.strategy, contractGroupName)
```

Reply:
Oh lol good catch